omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -117,6 +117,17 @@
 default['private_chef']['rabbitmq']['env_path'] = "/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/bin:/bin"
 
 ####
+# External RabbitMQ
+####
+default['private_chef']['external-rabbitmq']['enable'] = false
```

Comment:
This is similar to the pattern that has been proposed for the external postgresql, but I'm still feeling like it might be unnecessary to have the user both set down all the config for the external rabbit and flip this switch.  Since we control the the parsing of the configuration file, we should be able to distinguish between these two situations:

1) You've specified a rabbit configuration in chef-server.rb,
2) You have not specified a rabbit configuration in chef-server.rb

which is all we need to know to determine whether to set the action's rabbit config to the local rabbit or whatever the user provided.